chrome/content/zotero-better-bibtex/schomd.coffee

```diff
@@ -82,9 +82,13 @@ Zotero.BetterBibTeX.schomd.init = ->
     # \usepackage{xltxtra}, \textsuperscript{} \textsubscript{}
     text_escape: (text) ->
       text = '' unless text?
-      text = text.replace(/(\. )/g, ".\\ ")
-      text = text.replace(Zotero.CiteProc.CSL.SUPERSCRIPTS_REGEXP, ((aChar) -> "{\\textsuperscript{#{Zotero.CiteProc.CSL.SUPERSCRIPTS[aChar]}}}"))
+      savetext = text
+      text = text.replace(/([$_^%{&])/g, "\\$1")
```

Comment:
Huh. Never really thought about this. I suppose I could move Text2LeTeX out of the translators so it could be callable here.